td-layout/src/runtime.rs

```diff
@@ -44,6 +44,7 @@ pub const TD_PAYLOAD_SHADOW_STACK_SIZE: u32 = 0x200000;
 pub const TD_PAYLOAD_STACK_SIZE: u32 = 0x800000;
 
 pub const TD_PAYLOAD_PAGE_TABLE_BASE: u64 = 0x800000;
+pub const TD_PAGE_TABLE_SIZE: usize = 0x800000;
```

Comment:
Should it be named as "TD_PAYLOAD_PAGE_TABLE_SIZE"?

Reply:
Yes it should be. I'll update the patch, thanks